gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -96,9 +102,17 @@ public CommandTree(Composite parent, Models models, Widgets widgets) {
     SearchBox search = new SearchBox(this, false);
     loading = LoadablePanel.create(this, widgets, p -> new Tree(p, models, widgets));
     tree = loading.getContents();
+    commandIdx = createLabel(this, COMMAND_INDEX_DSCRP);
+    commandIdx.setToolTipText(COMMAND_INDEX_HOVER);
+    commandIdx.addListener(SWT.MouseDoubleClick, e -> {
+      if (commandIdx.getText().length() > COMMAND_INDEX_DSCRP.length()) {
+        widgets.copypaste.setContents(commandIdx.getText().substring(COMMAND_INDEX_DSCRP.length()));
```

Comment:
Is the substring necessary? It was created with COMMAND_INDEX_DSCRP just a few lines before.

Reply:
Yes it would be necessary, it will cut out the description part in the label string.
e.g. for a label string of `Command index: 123.0.1`, after using substring to clip it, we only send `123.0.1` to user's clipboard, so it's more ready to be pasted in the goto-command dialog.